src/entity/Guild/index.ts

```diff
@@ -2,3 +2,8 @@ export * from './Guild'
 export * from './GuildSettings'
 export * from './GuildSuggestion'
 export * from './GuildSupportTicket'
+export * from './GuildUser'
```

Comment:
Ok, looks like you've already added the classes to this file, so you just need to combine the import statements in the other files.